doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -47,6 +47,8 @@ This error could be caused by the following issues:
 
   * An out-of-tree external flash driver is not selecting :kconfig:option:`CONFIG_PM_EXTERNAL_FLASH_HAS_DRIVER`, resulting in partitions for images located in the external flash memory being not accessible.
 
+  * Insufficient value set for ``CONFIG_BOOT_MAX_IMG_SECTORS``, as MCUboot typically increases slot sizes when external flash is enabled.
```

Comment:
Given that this section has 4 bullets, of which 2 reference Kconfig options in the NCS, 2 reference Kconfig options elsewhere (MCUboot?), I suggest you clarify where these "elsewhere" Kconfigs can be found or set.